docs/pages/foundations/animation/implementation.js

```diff
@@ -0,0 +1,47 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex } from 'gestalt';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+import MainSection from '../../../docs-components/MainSection.js';
+import SandpackExample from '../../../docs-components/SandpackExample.js';
+import useExample from '../../../examples/animation/useExample.js';
+
+export default function AnimationImplementation(): Node {
+  return (
+    <Page title="Implementing animations">
+      <PageHeader
+        badge="pilot"
+        name="Implementing animations"
+        type="guidelines"
+        description="There are multiple ways to implement animations in product."
+      />
+      <MainSection
+        name="Gestalt"
+        description="Many of our components come with animations baked in! If you are implementing a custom animation, be sure to wrap the animation in the [useReducedMotion hook](https://gestalt.pinterest.systems/web/utilities/usereducedmotion) to ensure an accessible experience. This hook will automatically disable any contained animations when the user enables the Reduced Motion setting on their device."
+      />
+      <MainSection
+        name="CSS and InlineStyle"
+        description={`
+        Animations can be implemented using CSS via the InlineStyle package. See [this diff for an example](https://sourcegraph.pinadmin.com/phabricator.pinadmin.com/diffusion/P/pinboard/-/blob/webapp/app/common/react/components/WebMentorship/mentees/annisa/Profile.js?L24:5).
```

Comment:
I wouldn't recommend linking to a Web Mentorship file, as those aren't intended to be production-ready. There are plenty of other examples in the actual app we can use. Also, we should probably obfuscate the link behind a pinch link. (And we're not linking to a diff, but a file)

Also: InlineStyle isn't really a package (like an npm package), it's just a component. If we're linking to a usage in a file, we may want to link to [the component itself](https://phabricator.pinadmin.com/diffusion/P/browse/master/webapp/app/packages/InlineStyle.js).